pkgs/development/python-modules/hy/default.nix

```
@@ -4,31 +4,43 @@
 , colorama
 , fetchFromGitHub
 , funcparserlib
+, hy
 , pytestCheckHook
+, python
 , pythonOlder
 , rply
+, testers
+, toPythonApplication
+, hyDefinedPythonPackages ? python-packages: [ ] /* Packages like with python.withPackages */
 }:
 
 buildPythonPackage rec {
   pname = "hy";
-  version = "1.0a3";
+  version = "1.0a4";
 
```

```suggestion
format = "setuptools";

```

A minor nitpick, but would be helpful for maintenance.